### DISCUSSION TITLE
Flex config, vertical mode not showing preview

### DISCUSSION THREAD

**Discussion by benfrain** [nvim-telescope/telescope.nvim#1428] [Q&A] [answered]
Trying out flex layout but the vertical mode does not show a preview - horizontal shows preview as expected.

I call telescope like this:

```
map("n", "<leader>p", '<cmd>lua require("telescope.builtin").find_files()<cr>')
```

And the relevant portion of the setup looks like this (tried a few variations):

```
require("telescope").setup({
  defaults = {
    winblend = 20,
    sorting_strategy = "descending",
    layout_strategy = "flex",
    layout_config = {
      flex = {
        flip_columns = 80,
      },
      vertical = {
        width = 0.8,
        height = 0.8,
        preview_cutoff = 40,
        prompt_position = "bottom",
      },
      horizontal = {
        width = 0.8,
        height = 0.8,
      },
    },
...
```

Am I doing something wrong or is this possibly a bug?

**Comment by benfrain** [comment]
Further experimentation revealed that if I went narrow enough the preview appeared. 

Seems that if I remove the `width` and `height` for the vertical settings if stays there the whole time.

**Comment by l-kershaw** [comment, marked as answer]
Since your `flip_columns` (80) is less than the default `preview_cutoff` for `horizontal` (120), there is a range of window sizes where the horizontal strategy will disable the preview (and thus look similar to the vertical one). You can avoid this issue by changing your `preview_cutoff` for `horizontal` to a smaller value.

**Reply by benfrain** [reply]
Thanks! 👍
